tests/integrations/tso/client_test.go

```diff
@@ -278,32 +272,32 @@ func TestMixedTSODeployment(t *testing.T) {
 		}
 		cancel1()
 	}()
-	checkTSO(ctx1, re, &wg, backendEndpoints)
 	wg.Wait()
 }
 
 func checkTSO(ctx context.Context, re *require.Assertions, wg *sync.WaitGroup, backendEndpoints string) {
+	wg.Add(tsoRequestConcurrencyNumber)
 	for i := 0; i < tsoRequestConcurrencyNumber; i++ {
 		go func() {
 			defer wg.Done()
-			cli := mcs.SetupClientWithKeyspace(context.Background(), re, strings.Split(backendEndpoints, ","))
+			cli := mcs.SetupClientWithKeyspace(ctx, re, strings.Split(backendEndpoints, ","))
 			var ts, lastTS uint64
 			for {
-				physical, logical, err := cli.GetTS(context.Background())
-				// omit the error check since there are many kinds of errors
-				if err == nil {
-					ts = tsoutil.ComposeTS(physical, logical)
-					re.Less(lastTS, ts)
-					lastTS = ts
-				}
 				select {
 				case <-ctx.Done():
-					physical, logical, _ := cli.GetTS(context.Background())
-					ts = tsoutil.ComposeTS(physical, logical)
-					re.Less(lastTS, ts)
+					// Make sure the lastTS is not empty
+					re.NotEmpty(lastTS)
```

Comment:
Why not compare again?

Reply:
`re.Less` below is to do the check, here I just want to make sure the `lastTS` is set at least once.